examples/problem_decomposition/mifno.ipynb

```diff
@@ -230,21 +234,33 @@
      "output_type": "stream",
      "text": [
       "(All the energy values are in hartree)\n",
-      "Total MI-FNO energy = -15.595118283461277\n",
-      "Correlation energy = -0.03501990246337883\n",
-      "Mean-field energy = -15.560098380997898\n",
-      "        energy_total  energy_correlation  correction   epsilon\n",
-      "(0,)      -15.560249           -0.000150   -0.000150 -0.000150\n",
-      "(1,)      -15.576167           -0.016069   -0.000038 -0.016069\n",
-      "(2,)      -15.576167           -0.016069   -0.000038 -0.016069\n",
-      "(0, 1)    -15.576410           -0.016312   -0.000097 -0.000092\n",
-      "(0, 2)    -15.576410           -0.016312   -0.000097 -0.000092\n",
-      "(1, 2)    -15.594784           -0.034685    0.000000 -0.002547\n"
+      "Total incremental energy = -15.83647358459995\n",
+      "Correlation energy = -0.06915435696141081\n",
+      "Mean-field energy = -15.76731922763854\n",
+      "        energy_total  energy_correlation   epsilon  correction  n_electrons  \\\n",
```

Comment:
There is so interesting stuff in there (remember how many questions the guys from IBM asked us last year). Using a list of dots afterwards instead of a sentence, take the time to highlight what the quantities of interest mean. In particular I see you have chosen to show the number of electrons and spin orbitals: it means it is useful to you and your insight is welcome !